src/pydata_sphinx_theme/assets/styles/variables/_color.scss

```diff
@@ -5,97 +5,160 @@
+    "50": #fbf8fe,
+    "100": #f6f2fe,
+    "200": #ece3fc,
+    "300": #decdfa,
+    "400": #b691f5,
+    //WCAG AA compliant against light bg
+    "500": #894def,
+    "600": #6730c5,
+    "700": #4e2595,
+    "800": #31175f,
+    "900": #1e0e39,
+  ),
+  "info": (
+    "light": #276be9,
+    "light-bg": #eff4fd,
+    "dark": #79a3f2,
+    "dark-bg": #06245d,
+  ),
+  "warning": (
+    "light": #f66a0a,
+    "light-bg": #fff8f2,
+    "dark": #ff9245,
+    "dark-bg": #652a02,
+  ),
   "success": (
-    "light": rgb(40, 167, 69),
-    "dark": rgb(72, 135, 87),
+    "light": #00843f,
+    "light-bg": #ecf6f1,
+    "dark": #5fb488,
+    "dark-bg": #002f17,
   ),
   "attention": (
-    "light": rgb(255, 193, 7),
-    "dark": rgb(220, 169, 15),
+    "light": #f66a0a,
+    "light-bg": #fff8f2,
+    "dark": #ff9245,
+    "dark-bg": #652a02,
   ),
   "danger": (
-    "light": rgb(220, 53, 69),
-    "dark": rgb(203, 70, 83),
+    "light": #d72d47,
+    "light-bg": #fcf1f2,
+    "dark": #e78894,
+    "dark-bg": #4e111b,
   ),
   "text-base": (
-    "light": rgb(50, 50, 50),
-    // Black + 50
-    "dark": rgb(206, 206, 206),
-    // White - 50
+    "light": rgb(34, 40, 50),
+    // gray-800
+    // "dark": #e5e7ea
+    "dark": #ced6dd,
+    // gray-100
   ),
   "text-muted": (
-    "light": rgb(100, 100, 100),
-    // Twice as far from 0 as base
-    "dark": rgb(166, 166, 166),
-    // Twice as far from 256 as base
-  ),
-  "shadow": (
-    "light": rgb(216, 216, 216),
-    "dark": rgb(33, 33, 33),
-  ),
-  "border": (
-    "light": rgb(201, 201, 201),
-    "dark": rgb(192, 192, 192),
-  ),
-  "inline-code": (
-    "light": rgb(232, 62, 140),
-    "dark": rgb(221, 158, 194),
-  ),
-  "target": (
-    "light": rgb(251, 229, 78),
-    "dark": rgb(71, 39, 0),
+    "light": #48566b,
+    // gray-600
+    "dark": #d1d5da,
+    // gray-300
   ),
+  // TODO - update
+  "shadow":
+    (
+      // "light": rgb(216, 216, 216),
+      "light": rgba(0, 0, 0, 0.1),
+      "dark": rgba(0, 0, 0, 0.2),
+    ),
+  // TODO- update (need muted and regular - this is muted)
+  "border":
+    (
+      "light": rgba(23, 23, 26, 0.2),
+      "dark": #48566b,
+    ),
+  // TODO - update
+  "inline-code":
+    (
+      "light": rgb(232, 62, 140),
+      "dark": rgb(221, 158, 194),
+    ),
+  // TODO - update
+  "target":
+    (
+      "light": rgb(251, 229, 78),
+      "dark": rgb(71, 39, 0),
+    ),
   // DEPTH COLORS
   // background: color of the canvas / the furthest back layer
   "background":
     (
       "light": rgb(255, 255, 255),
-      "dark": rgb(18, 18, 18),
+      "dark": #14181e,
     ),
   // on-background: provides slight contrast against background
   // (by use of shadows in light theme)
   "on-background":
     (
       "light": rgb(255, 255, 255),
-      "dark": rgb(30, 30, 30),
+      "dark": #222832,
+      // gray-800
     ),
   // surface: object set above the background (without shadows)
   // Uses JupyterLab cell background colors in light/dark theme
```

Comment:
comment on this line is probably obsolete since we're about to change the colors.